main/test/ca/uwaterloo/flix/language/phase/TestParser.scala

```diff
@@ -731,6 +731,87 @@ class TestParser extends FunSuite with TestUtils {
     run(input)
   }
 
+  test("Expression.Array.01") {
+    val input = "def f(): Unit = [||]"
+    run(input, core = false)
+
+  }
+
+  test("Expression.ArrayLit.02") {
+    val input = "def f(): Int = [|42|] "
+    run(input, core = false)
+  }
+
+  test("Expression.ArrayLit.03") {
+    val input = "def f(): Int = [|42, 21|]"
+    run(input, core = false)
+  }
+
+  test("Expression.ArrayLit.03") {
+    val input = "def f(): Int = [|42, x, 21|]"
```

Comment:
The problem with a test like this is that `x` is undefined so it will fail to during the Resolver.

All tests should be valid programs that can compile and produce a result.